pom.xml

```diff
@@ -320,6 +320,11 @@
       <artifactId>grizzled-slf4j_${scala.version.prefix}</artifactId>
       <version>1.0.3</version>
     </dependency>
+
+    <!--
+      Only used directly in tests, but adding test-scope blocks transitive inclusion via Spark, resulting in runtime
```

Comment:
If Hadoop needs commons-io, then it should depend on it; lo and behold, it does:

```
$ mvn dependency:tree -Dverbose -Dincludes="commons-io:" | perl -pe 's/^\[INFO\] //'
…
org.hammerlab.guacamole:guacamole:jar:0.0.1-SNAPSHOT
+- org.bdgenomics.adam:adam-core_2.10:jar:0.19.0:compile
|  \- (commons-io:commons-io:jar:1.3.2:compile - omitted for conflict with 2.5)
+- org.apache.spark:spark-core_2.10:jar:1.6.1:compile
|  \- org.tachyonproject:tachyon-client:jar:0.8.2:compile
|     \- (commons-io:commons-io:jar:2.4:compile - omitted for conflict with 1.3.2)
+- org.apache.hadoop:hadoop-client:jar:2.7.0:compile
|  +- org.apache.hadoop:hadoop-common:jar:2.7.0:compile
|  |  \- (commons-io:commons-io:jar:2.4:compile - omitted for conflict with 1.3.2)
|  +- org.apache.hadoop:hadoop-hdfs:jar:2.7.0:compile
|  |  \- (commons-io:commons-io:jar:2.4:compile - omitted for conflict with 1.3.2)
|  \- org.apache.hadoop:hadoop-mapreduce-client-core:jar:2.7.0:compile
|     \- org.apache.hadoop:hadoop-yarn-common:jar:2.7.0:compile
|        \- (commons-io:commons-io:jar:2.4:compile - omitted for conflict with 1.3.2)
+- commons-io:commons-io:jar:2.5:compile
\- com.holdenkarau:spark-testing-base_2.10:jar:1.6.1_0.4.4:test (scope not updated to compile)
   +- org.apache.hadoop:hadoop-hdfs:jar:tests:2.6.0:test
   |  \- (commons-io:commons-io:jar:2.4:test - omitted for conflict with 2.5)
   +- org.apache.hadoop:hadoop-common:jar:tests:2.6.0:test
   |  \- (commons-io:commons-io:jar:2.4:test - omitted for conflict with 2.5)
   +- org.apache.hadoop:hadoop-yarn-server-tests:jar:tests:2.6.0:test
   |  \- org.apache.hadoop:hadoop-yarn-server-resourcemanager:jar:2.6.0:test
   |     \- (commons-io:commons-io:jar:2.4:test - omitted for conflict with 2.5)
   \- org.apache.hadoop:hadoop-minicluster:jar:2.6.0:test
      +- org.apache.hadoop:hadoop-common:test-jar:tests:2.6.0:test
      |  \- (commons-io:commons-io:jar:2.4:test - omitted for conflict with 2.5)
      \- org.apache.hadoop:hadoop-hdfs:test-jar:tests:2.6.0:test
         \- (commons-io:commons-io:jar:2.4:test - omitted for conflict with 2.5)
…
```

(This is our dep tree before this PR).

Our top-level dep on commons-io:2.5 was apparently a kludge to resolve the conflict between hadoop and spark's commons-io:2.4 dep with adam-core's commons-io:1.3.2 dep.

The right way to resolve that is to exclude adam-core from bringing in its older version and competing with spark and hadoop; afaict ADAM runs fine with 2.4, but hadoop and spark miss `Charsets` (added in 2.3) when 1.3.2 wins.

I've now done that:

```
org.hammerlab.guacamole:guacamole:jar:0.0.1-SNAPSHOT
+- org.apache.spark:spark-core_2.10:jar:1.6.1:compile
|  \- org.tachyonproject:tachyon-client:jar:0.8.2:compile
|     \- commons-io:commons-io:jar:2.4:compile
+- org.apache.hadoop:hadoop-client:jar:2.7.0:compile
|  +- org.apache.hadoop:hadoop-common:jar:2.7.0:compile
|  |  \- (commons-io:commons-io:jar:2.4:compile - omitted for duplicate)
|  +- org.apache.hadoop:hadoop-hdfs:jar:2.7.0:compile
|  |  \- (commons-io:commons-io:jar:2.4:compile - omitted for duplicate)
|  \- org.apache.hadoop:hadoop-mapreduce-client-core:jar:2.7.0:compile
|     \- org.apache.hadoop:hadoop-yarn-common:jar:2.7.0:compile
|        \- (commons-io:commons-io:jar:2.4:compile - omitted for duplicate)
\- com.holdenkarau:spark-testing-base_2.10:jar:1.6.1_0.4.4:test (scope not updated to compile)
   +- org.apache.hadoop:hadoop-hdfs:jar:tests:2.6.0:test
   |  \- (commons-io:commons-io:jar:2.4:test - omitted for duplicate)
   +- org.apache.hadoop:hadoop-common:jar:tests:2.6.0:test
   |  \- (commons-io:commons-io:jar:2.4:test - omitted for duplicate)
   +- org.apache.hadoop:hadoop-yarn-server-tests:jar:tests:2.6.0:test
   |  \- org.apache.hadoop:hadoop-yarn-server-resourcemanager:jar:2.6.0:test
   |     \- (commons-io:commons-io:jar:2.4:test - omitted for duplicate)
   \- org.apache.hadoop:hadoop-minicluster:jar:2.6.0:test
      +- org.apache.hadoop:hadoop-common:test-jar:tests:2.6.0:test
      |  \- (commons-io:commons-io:jar:2.4:test - omitted for duplicate)
      \- org.apache.hadoop:hadoop-hdfs:test-jar:tests:2.6.0:test
         \- (commons-io:commons-io:jar:2.4:test - omitted for duplicate)
```